### ISSUE TITLE
Compile Fail

### ISSUE THREAD

**Issue by robshizzle** [nesbox/TIC-80#1020]
Compiling the latest commit yields...

```
/home/user/software/tic-80/src/console.c:787:52: error: ‘tic_sfx’ {aka ‘struct <anonymous>’} has no member named ‘waveform’; did you mean ‘waveforms’?
  787 |  {"WAVES",   WAVES_COUNT,   offsetof(tic_bank, sfx.waveform),  sizeof(tic_waveform),   true},
[ 74%] Building C object 3rd-party/sdl2/CMakeFiles/SDL2-static.dir/src/video/SDL_fillrect.c.o
make[2]: *** [CMakeFiles/tic80lib.dir/build.make:156: CMakeFiles/tic80lib.dir/src/console.c.o] Error 1
make[2]: Leaving directory '/home/user/software/tic-80/build'
make[1]: *** [CMakeFiles/Makefile2:475: CMakeFiles/tic80lib.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
```

**Comment by nesbox** [comment]
hmm, pls try a clean build, because on Travis I see all is ok
https://travis-ci.org/github/nesbox/TIC-80/builds

**Comment by robshizzle** [comment]
This issue is occurring with multiple users on Arch Linux. Multiple machines all producing the same error.

**Comment by nesbox** [comment]
Seems I got the problem, you are trying to build pro version.
Will fix it in 2 hours, thanks :)

**Comment by nesbox** [comment]
fixed here 4a8f421
